src/main/fc/core.c

```diff
@@ -524,11 +525,19 @@ void tryArm(void)
             if (isModeActivationConditionPresent(BOXFLIPOVERAFTERCRASH)) {
                 // Set motor spin direction
                 if (!(IS_RC_MODE_ACTIVE(BOXFLIPOVERAFTERCRASH) || (tryingToArm == ARMING_DELAYED_CRASHFLIP))) {
+                    if (cmpTimeUs(currentTimeUs, lastDisarmTimeUs) < 450000 && flipOverAfterCrashWasActiveLast) {
+                        tryingToArm = ARMING_DELAYED_NORMAL;
```

Comment:
I wonder if there is any point calculating 
```
const bool disarmedRecently = cmpTimeUs(currentTimeUs, lastDisarmTimeUs) < 450000;
```
before line 525, to simplify
```
if (cmpTimeUs(currentTimeUs, lastDisarmTimeUs) < 450000 && flipOverAfterCrashWasActiveLast) {
```
to
```
if (cmpTimeUs(disarmedRecently && flipOverAfterCrashWasActiveLast) {
```